src/components/application_manager/test/app_launch/app_launch_data_json_test.cc

```diff
@@ -102,7 +102,7 @@ class AppLaunchDataJsonTest : public ::testing::Test {
   void GetApplicationData_EXPECT_FALSE(const ApplicationData& in_data);
   std::string AddCounter(const std::string& inp, int32_t val);
 
-  std::auto_ptr<resumption::LastState> test_last_state_;
+  std::auto_ptr<resumption::LastStateImpl> test_last_state_;
```

Comment:
@dcherniev OK, but can you handle last state by Interface but not implementation?